### PR TITLE
updated link to vfio-app

### PR DESCRIPTION
**Pull request by stschmitt**
the filename of vfio-app.txt changed to .rst
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/Documentation/s390/vfio-ap.rst

### CODE REVIEW

**Comment by vmorris** [comment]
I would rather link to https://www.kernel.org/doc/html/latest/s390/vfio-ap.html

File: README.md [hunk 1]
```diff
@@ -411,7 +411,7 @@ device from the host through Qemu into the runq VM guest. VFIO mediated devices
 docker run --runtime runq -e RUNQ_APUUID=b34543ee-496b-4769-8312-83707033e1de ...
 ```
 For details on how to setup mediated devices on the host see
-https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/Documentation/s390/vfio-ap.txt
+https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/Documentation/s390/vfio-ap.rst
```
Comment (vmorris) [inline README.md:414]:
```suggestion
https://www.kernel.org/doc/html/latest/s390/vfio-ap.html
```

**Comment by stschmitt** [comment]
@vmorris Thanks. I was reading the doc over the weekend and saw the html version as well. Makes more sense to use the HTML version

**Comment by pmorjan** [comment]
LGTM. Thanks!

**Pull request merged**